pkg/executor/server.go

```diff
@@ -299,23 +311,21 @@ func (s *Server) serveStats(w http.ResponseWriter, req *http.Request) {
 		// Backward compatibility without uuid information
 		podFullName := kubelet.GetPodFullName(&api.BoundPod{
 			ObjectMeta: api.ObjectMeta{
-				Name: components[1],
-				// TODO(k8s): I am broken
+				Name:        components[1],
 				Namespace:   api.NamespaceDefault,
 				Annotations: map[string]string{kubelet.ConfigSourceAnnotationKey: s.sourcename},
 			},
 		})
 		stats, err = s.host.GetContainerInfo(podFullName, "", components[2], &query)
-	case 4:
+	case 5:
```

Comment:
k8s code, there is no spoon, I mean case 4. Valid URL paths handled by this target basically have 1, 3, or 5 path segments. Anything else is not supported.